communication/message_formatting.md

```diff
@@ -4,12 +4,23 @@ Message Formatting
 - **3.5.1** ![](/badge/req.png) A client must encode emoticons as Unicode emoji.
 
 - **3.5.2** ![](/badge/req.png) A receiving client will make a best-effort to display
-  text exactly as it was sent, and must not transform the text in any way.
+  text exactly as it was intended, and must not materially change the text in any way.
```

Comment:
>exactly as it was intended

It's not a client's job to read your mind. Re-word this back to "exactly as it was sent" and include exceptions where applicable. Also remove "materially change" and replace it with its definition.